app/models/search/goods_nomenclature_match.rb

```diff
@@ -12,7 +12,12 @@ class GoodsNomenclatureMatch < BaseMatch
     array_attr_writer :sections, :chapters, :headings, :commodities
 
     def commodities=(commodity_data)
-      @commodities = commodity_data.map { |cd| Commodity.new(cd['_source']) }
+      @commodities = commodity_data.map do |cd|
+        attributes = cd['_source']
+        attributes['score'] = cd['_score'] if cd['_score'].present?
+
+        Commodity.new(attributes)
+      end
```

Comment:
I believe that:

`attributes['score'] = cd['_score'] if cd['_score'].present?`
is the same as
`attributes['score'] = cd['_score']`

Reply:
if we want to remove the null cases, we could filter it out:
```
@commodities = commodity_data
                       .filter{|cd| cd['_score'].present?}
                       .map{...
```

Reply:
That's an additional iteration that doesn't look better/add any benefits really. I'm happy just not assigning for now